axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -622,17 +643,22 @@ def test_strategy(self):
         self.transitions_test(state_and_actions)
 
         # finished: 0, 5, 10
-        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C), (11, D), (5, D), (10, D), (8, C)] * 3
+        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C),
+                                                          (11, D), (5, D),
+                                                          (10, D), (8, C)] * 3
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 5, 10, 11, 12, 15
-        state_and_actions = ([(0, D), (12, C), (8, D), (5, D), (10, C), (11, C), (15, C), (15, C), (15, D)] +
+        state_and_actions = ([(0, D), (12, C), (8, D), (5, D),
+                              (10, C), (11, C), (15, C), (15, C), (15, D)] +
                              [(2, C)] * 3 + [(2, D), (14, C), (13, C)])
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 3, 5, 10, 11, 12, 13, 14, 15
         to_state_fourteen = [(0, D), (12, D), (11, C), (15, D), (2, D)]
-        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C), (13, D), (7, C)] + [(3, D), (3, C)] * 3
+        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C),
+                                                 (13, D), (7, C)] + [(3, D),
+                                                                     (3, C)] * 3
```

Comment:
I'm generally easy but do feel that calls for longer than 80 are not usually (I obviously don't know your brother's reasons) for great reasons: not everyone **does** have a wide monitor and actually think the 80 character limit is more inclusive. Some people work on small laptops and 80 still lets them have two windows up without wrapping :)

Either way: we've generally stuck to 80 in the library so would be great to stick to it :+1: